build/yaml/runUnitTests-CI.yml

```diff
@@ -0,0 +1,27 @@
+steps:
+- task: UseDotNet@2
+  displayName: 'Use .Net Core sdk 3.1.x'
+  inputs:
+    version: 3.1.x
+
+- task: DotNetCoreCLI@2
+  displayName: 'Build'
+  inputs:
+    command: build
+    publishWebProjects: false
+    projects: 'Tests/TranscriptTestRunnerTests/TranscriptTestRunnerTests.csproj'
+    arguments: '-v n --configuration $(BuildConfiguration)'
+
+- task: DotNetCoreCLI@2
+  displayName: 'DotNet Test'
+  inputs:
+    command: test
+    projects: 'Tests/TranscriptTestRunnerTests/TranscriptTestRunnerTests.csproj'
+    arguments: "-v n --configuration $(BuildConfiguration) --no-build --no-restore"
+
```

Comment:
Can we add a [PublishBuildArtifacts@1](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/publish-build-artifacts?view=azure-devops) task here to publish the trx as an artifact so it is easier to download it right from the build results page?
Set the artifactName property to testResults (rather than using the default drop name).

Reply:
As discussed yesterday, the result of the test can be downloaded from the pipeline's summary in Azure. We only set the _testRunTitle_ to a readable name.